aggregator/Dockerfile

```diff
@@ -1,3 +1,10 @@
-FROM openjdk:14
-COPY build/aggregator-all.jar .
-CMD java -jar aggregator-all.jar
+# first build the jar
+# assumses build context is the root of this project
+FROM gradle:jdk14 as builder
+COPY . .
+RUN gradle aggregator:shadowJar
+
+# run the built jar
+FROM openjdk:14 AS runner
+COPY --from=builder /home/gradle/aggregator/build/aggregator-all.jar .
+CMD java -jar aggregator-all.jar
```

Comment:
Does this not need a config file? Should we at least `touch` the config file so we can use the launch shorthand?

Reply:
```
RUN touch /root/config.yml
ENV HYPERION_AGGREGATOR_CONFIG=/root/config.yml
```